awx/main/tasks/receptor.py

```diff
@@ -487,9 +494,7 @@ def receptor_params(self):
 
     @property
     def sign_work(self):
-        if self.work_type in ('ansible-runner', 'local'):
-            return work_signing_enabled(self.config_data)
-        return False
+        return work_signing_enabled(self.receptor_ctl, self.execution_node, self.work_type)
```

Comment:
We still don't intend to sign work for container_group jobs, right? The keys would have to be backed into the image or something and that's not workable.

Container group jobs don't use the "ansible-runner" work type, so that's how this returned False in that case. I think we still want to return False in that case.